OrbitGl/App.cpp

```diff
@@ -1173,5 +1173,6 @@ void OrbitApp::InitializeClientTransactions() {
 
 //-----------------------------------------------------------------------------
 void OrbitApp::FilterFunctions(const std::string& filter) {
-  m_LiveFunctionsDataView->OnFilter(filter);
+  Capture::GFunctionFilter = filter;
```

Comment:
Maybe just add a getter to OrbitApp which in turn asks m_LiveFunctionDataView for the string instead of introducing a global?

Reply:
Yeah, I thought about that, but to me the filter relates to the capture and not the app in general.  Eventually we should remove all static/global members from the Capture class, this way we could have multiple captures loaded at once with their individual filters.

Reply:
But this could probably be cleaned up I agree.  Will address in other PR as this is bugfix I'd want in before code freeze.